webplugin/js/app/kommunicate-ui.js

```
@@ -292,131 +258,100 @@ KommunicateUI = {
-            data.appId = kommunicate._globals.appId;
+    initFaq: function () {
+        var data = {};
+        data.appId = kommunicate._globals.appId;
 
         // On Click of FAQ button the FAQ category List will open.
         $applozic(d).on('click', '#km-faq', function () {
             var isFaqCategoryPresent =
-                kommunicate &&
-                kommunicate._globals &&
-                kommunicate._globals.faqCategory;
+                kommunicate && kommunicate._globals && kommunicate._globals.faqCategory;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)